app/src/main/java/co/nimblehq/compose/crypto/ui/screens/navigation/CryptoNavGraph.kt

```diff
@@ -0,0 +1,38 @@
+package co.nimblehq.compose.crypto.ui.screens.navigation
+
+import androidx.activity.ComponentActivity
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import co.nimblehq.compose.crypto.extension.provideViewModels
+import co.nimblehq.compose.crypto.ui.screens.home.HomeScreen
+import co.nimblehq.compose.crypto.ui.screens.home.HomeViewModel
+import co.nimblehq.compose.crypto.ui.screens.information.CoinInfoScreen
+
+@Suppress("FunctionNaming")
+@Composable
+fun CryptoNavGraph(
+    componentActivity: ComponentActivity,
+    navController: NavHostController = rememberNavController(),
+    startDestination: String = CryptoScreen.HOME.name
+) {
+    NavHost(
+        navController = navController,
+        startDestination = startDestination
+    ) {
+        composable(CryptoScreen.HOME.name) {
+            val homeViewModel: HomeViewModel by componentActivity.provideViewModels()
+            HomeScreen(
+                viewModel = homeViewModel,
+                onMyCoinsItemClick = {
+                    navController.navigate(route = CryptoScreen.COIN_INFORMATION.name)
+                }
+            )
+        }
+        composable(CryptoScreen.COIN_INFORMATION.name) {
+            CoinInfoScreen()
+        }
+    }
+}
```

Comment:
@luongvo @Wadeewee Let' keep it there 😄 I will refactor the navigation graph on my integration task.

Reply:
> I think you did overwork here, you just half finished the task https://github.com/nimblehq/jetpack-compose-crypto/issues/19. This is a UI task, a screen implementation is enough, no nav graph and events handling are needed 😛

@luongvo If we don't handle some navigation part in this ticket, We need to set the default  screen as `CoinInfoScreen` instead of `HomeScreen` and I'm afraid It will be conflict when merge and rebase that's why I add it in this PR.

> @luongvo @Wadeewee Let' keep it there 😄 I will refactor the navigation graph on my integration task.

@hoangnguyen92dn I will leave todo that's we need to update it then 🚀

Reply:
> I'm afraid It will be conflict when merge and rebase that's why I add it in this PR.

@Wadeewee you can just 
- take care about the screen UI, leave the rest back.
- you can change the default startup screen but don't push it up.
- add a note about this, how to show the new UI.

I think for UI ticket, we can just do that in case it requires complex logic for navigation from Integration task cc @hoangnguyen92dn